cluster/gce/gci/configure-helper.sh

```diff
@@ -1858,6 +1858,17 @@ function prepare-etcd-manifest {
   fi
   # Replace the volume host path.
   sed -i -e "s@/mnt/master-pd/var/etcd@/mnt/disks/master-pd/var/etcd@g" "${temp_file}"
+  # Replace the runAsUser and runAsGroup
+  sed -i -e "s@{{runAsUser}}@${ETCD_RUNASUSER:-0}@g" "${temp_file}"
+  sed -i -e "s@{{runAsGroup}}@${ETCD_RUNASGROUP:-0}@g" "${temp_file}"
+  chown -R ${ETCD_RUNASUSER:-0}:${ETCD_RUNASGROUP:-0} /var/etcd/
+  # Replace capabilities
+  if [[ -n "${ETCD_RUNASUSER:-}" && -n "${ETCD_RUNASGROUP:-}" ]]; then
+    container_security_context=$(echo "{\"allowPrivilegeEscalation\": false, \"capabilities\": {\"drop\": [\"all\"]}}")
+  else
+    container_security_context=$(echo "{}")
```

Comment:
I think instead of making the `securityContext: {}` when ETCD_RUNASUSER is not set, you can make it so that:-
instead of having 
```
securityContext: {{security_context}}
```
You make it just:
```
{{containerSecurityContext}}
```

Then this would be
```
container_security_context=""
if [[ -n "${ETCD_RUNASUSER:-}" && -n "${ETCD_RUNASGROUP:-}" ]]; then
    container_security_context=$(echo "{\"allowPrivilegeEscalation\": false, \"capabilities\": {\"drop\": [\"all\"]}}")
fi
sed -i -e "s@{{security_context}}@${container_security_context}@g" "${temp_file}"
```

Reply:
fixed.